src/services/db/RepoService.ts

```diff
@@ -1,5 +1,7 @@
+import { User } from "aws-sdk/clients/budgets"
```

Comment:
nit: I think this is not needed

Reply:
Thanks for the catch! Removed in ce3615eb571009f5f1bf3074191def1847dfa87b